src/main/java/net/minecraftforge/client/event/RenderSpecificHandEvent.java

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
Would it make sense to rename the hand event to just RenderHandEvent, since it's the only one now?